### PR TITLE
[BD-6] Use python 3.8 in hermes role

### PR DESCRIPTION
**Pull request by ericfab179**
Add deadsnakes PPA and install python 3.8 package in Xenial and Bionic.
Use Python 3.8 to create hermes virtualenv.

**Reviewers**
- [ ] @awais786 
- [ ] @morenol

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @ericfab179! I've created [BLENDED-515](https://openedx.atlassian.net/browse/BLENDED-515) to keep track of it in Jira.   More details are on the [BD-06](https://openedx.atlassian.net/wiki/spaces/COMM/pages/1594000004/BD-06+Python+3.8+Upgrade) project page.


When this pull request is ready, tag your edX technical lead.

File: playbooks/roles/hermes/tasks/main.yml [hunk 1]
```diff
@@ -20,29 +20,26 @@
...
 #
 
-# The deadsnakes PPA is required to install python3.6 on Xenial.
-# Bionic comes with python3.6 installed.
+# The deadsnakes PPA is required to install python3.8 on Xenial and Bionic.
 - name: add deadsnakes repository
   apt_repository:
-    repo: "ppa:fkrull/deadsnakes"
-  when: ansible_distribution_release == 'xenial'
+    repo: "ppa:deadsnakes/ppa"
```
Comment (morenol) [inline playbooks/roles/hermes/tasks/main.yml:26]: In this case, this is only needed if the distribution is not ubuntu focal

**Comment by awais786** [comment]
Already implemented. Thanks for creating this PR.

**Comment by openedx-webhooks** [comment]
@ericfab179 Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/configuration&entry.867055334=https://github.com/openedx/configuration/pull/5889&entry.1484655318=https://github.com/ericfab179&entry.752974735=2020-07-25+03:12&entry.1917517419=2022-08-19+06:50&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**